File: experimento_2/output/gemini-3.1-pro-preview/zero/cpp/Relógio.cpp

#include <iostream>

using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    long long H, M, S, T;
    if (cin >> H >> M >> S >> T) {
        long long total_seconds = H * 3600 + M * 60 + S + T;
        
        long long new_S = total_seconds % 60;
        long long new_M = (total_seconds / 60) % 60;
        long long new_H = (total_seconds / 3600) % 24;
        
        cout << new_H << "\n" << new_M << "\n" << new_S << "\n";
    }

    return 0;
}

File: output/deepseek_deepseek_v3_2/few/python_noimg/Relógio.py
import sys

def main():
    data = sys.stdin.read().strip().split()
    if not data:
        return
    
    H = int(data[0])
    M = int(data[1])
    S = int(data[2])
    T = int(data[3])
    
    total_seconds = H * 3600 + M * 60 + S + T
    
    total_seconds %= 24 * 3600
    
    new_H = total_seconds // 3600
    total_seconds %= 3600
    new_M = total_seconds // 60
    new_S = total_seconds % 60
    
    print(new_H)
    print(new_M)
    print(new_S)

if __name__ == "__main__":
    main()